### PR TITLE
ci: remove sync action

### PR DESCRIPTION
**Pull request by kayman233**
:baby_chick: Download canary assets:

[colors--canary.320.4123273032.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/colors--canary.320.4123273032.xml)  
[color_metro_ios-swift--canary.320.4123273032.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_ios-swift--canary.320.4123273032.swift)  
[color_metro_kotlin--canary.320.4123273032.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_kotlin--canary.320.4123273032.kt)  
[color_metro_react-native--canary.320.4123273032.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_react-native--canary.320.4123273032.ts)  
[color_sbermarket_ios-swift--canary.320.4123273032.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_ios-swift--canary.320.4123273032.swift)  
[color_sbermarket_kotlin--canary.320.4123273032.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_kotlin--canary.320.4123273032.kt)  
[color_sbermarket_react-native--canary.320.4123273032.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_react-native--canary.320.4123273032.ts)  
[color_sberprime_ios-swift--canary.320.4123273032.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_ios-swift--canary.320.4123273032.swift)  
[color_sberprime_kotlin--canary.320.4123273032.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_kotlin--canary.320.4123273032.kt)  
[color_sberprime_react-native--canary.320.4123273032.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_react-native--canary.320.4123273032.ts)  
[color_selgros_ios-swift--canary.320.4123273032.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_ios-swift--canary.320.4123273032.swift)  
[color_selgros_kotlin--canary.320.4123273032.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_kotlin--canary.320.4123273032.kt)  
[color_selgros_react-native--canary.320.4123273032.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_react-native--canary.320.4123273032.ts)  
[color_smbusiness_ios-swift--canary.320.4123273032.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_ios-swift--canary.320.4123273032.swift)  
[color_smbusiness_kotlin--canary.320.4123273032.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_kotlin--canary.320.4123273032.kt)  
[color_smbusiness_react-native--canary.320.4123273032.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_react-native--canary.320.4123273032.ts)  
[PlasmaTokensColor--canary.320.4123273032.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/PlasmaTokensColor--canary.320.4123273032.swift)  
[shadow_sbermarket_react-native--canary.320.4123273032.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_react-native--canary.320.4123273032.ts)  
[typo_mage_ios-swift--canary.320.4123273032.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_ios-swift--canary.320.4123273032.swift)  
[typo_mage_kotlin--canary.320.4123273032.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_kotlin--canary.320.4123273032.kt)  
[typo_mage_react-native--canary.320.4123273032.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_react-native--canary.320.4123273032.ts)  
[typo_plasma_ios-swift--canary.320.4123273032.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_ios-swift--canary.320.4123273032.swift)  
[typo_plasma_kotlin--canary.320.4123273032.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_kotlin--canary.320.4123273032.kt)  
[typo_plasma_react-native--canary.320.4123273032.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_react-native--canary.320.4123273032.ts)  
[typo_ruler_ios-swift--canary.320.4123273032.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_ios-swift--canary.320.4123273032.swift)  
[typo_ruler_kotlin--canary.320.4123273032.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_kotlin--canary.320.4123273032.kt)  
[typo_ruler_react-native--canary.320.4123273032.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_react-native--canary.320.4123273032.ts)  
[typo_sage_ios-swift--canary.320.4123273032.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_ios-swift--canary.320.4123273032.swift)  
[typo_sage_kotlin--canary.320.4123273032.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_kotlin--canary.320.4123273032.kt)  
[typo_sage_react-native--canary.320.4123273032.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_react-native--canary.320.4123273032.ts)  
[typo_sbermarket_ios-swift--canary.320.4123273032.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_ios-swift--canary.320.4123273032.swift)  
[typo_sbermarket_kotlin--canary.320.4123273032.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_kotlin--canary.320.4123273032.kt)  
[typo_sbermarket_react-native--canary.320.4123273032.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_react-native--canary.320.4123273032.ts)  
[typo_soulmate_ios-swift--canary.320.4123273032.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_ios-swift--canary.320.4123273032.swift)  
[typo_soulmate_kotlin--canary.320.4123273032.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_kotlin--canary.320.4123273032.kt)  
[typo_soulmate_react-native--canary.320.4123273032.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_react-native--canary.320.4123273032.ts)

### CODE REVIEW

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **💀 Missing report, something went wrong**

Check out current report in job artifacts!

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-320/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-320/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out comparison report in job artifacts!

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-320/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-320/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-320/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-320/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-320/temple-storybook/

File: .github/workflows/documentation-pr.yml [hunk 1]
```diff
@@ -106,17 +106,16 @@ jobs:
           remote: "/var/www/plasma"
 
       - name: s3 Upload builds
-        uses: jakejarvis/s3-sync-action@master
-        with:
-          args: --acl public-read --follow-symlinks --delete
+        run: >
+          aws s3 sync
+          ./s3_build/${PR_NAME}/
+          s3://${{ secrets.AWS_S3_BUCKET_2 }}/pr/${PR_NAME}/
```
Comment (Yeti-or) [inline .github/workflows/documentation-pr.yml:112]: слеш в конце появился осознанно?

File: .github/workflows/documentation-main.yml [hunk 2]
```diff
@@ -108,14 +108,13 @@ jobs:
             mv next-${SHA} current
 
       - name: s3 Upload documentation build
-        uses: jakejarvis/s3-sync-action@master
-        with:
-          args: --acl public-read --follow-symlinks --delete
+        run: >
+          aws s3 sync
+          ./s3_build/next-${{ github.sha }}/
+          s3://${{ secrets.AWS_S3_BUCKET_2 }}/current/
```
Comment (Yeti-or) [inline .github/workflows/documentation-main.yml:114]: слэш нужен ?

File: .github/workflows/examples-pr.yml [hunk 3]
```diff
@@ -69,17 +69,16 @@ jobs:
           remote: "/var/www/plasma"
 
       - name: s3 Upload build
-        uses: jakejarvis/s3-sync-action@master
-        with:
-          args: --acl public-read --follow-symlinks --delete
+        run: >
+          aws s3 sync
+          ./s3_build/demo-canvas-app-${PR_NAME}/
+          s3://${{ secrets.AWS_S3_BUCKET_2 }}/pr/demo-canvas-app-${PR_NAME}/
```
Comment (Yeti-or) [inline .github/workflows/examples-pr.yml:75]: тот же вопрос

**Review by Yeti-or — APPROVED** [review]

**Pull request merged**